ea-2021-04-ndvi-automation-brian-pinke.py

```diff
@@ -0,0 +1,790 @@
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# A higher NDVI value generally indicates more vegetation, and a lower value generally indicates less vegetation. In the NDVI plot, HARV has a higher mean NDVI, above 0.8, from mid May to late September. This correlates well with the warmer and rainy season of MA and would be the good period of time to fly. SJER has a higher NDVI, above 0.65, between mid February and April, staying above 0.6 through the end of April. This correlates well with the peak growing season of the area in early March, and would be a good period of time to fly.
+
+# # Question 2 (10 points)
+# 
+# How could you modify your workflow to look at vegetation changes over time in each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# The workflow here captures mean ndvi over the course of a single year. If data were provided for multiple years, this workflow would loop through the additional directory paths and capture the NDVI for all dates provided. The plot could then be adapted to show vegetation changes over time by plotting different line colors for each year, with constant symbols for each location.
+# 
+# Another way to capture vegetation change over time would be to groupby mean NDVI for the full year, and plot Mean NDVI per scene by year, rather than by month.
+# 
+
+# # Do not edit this cell! (10 points)
+# 
+# The notebook includes:
+# * additional Markdown cells throughout the notebook to describe: 
+#     * the data that you used - and where it is from
+#     * how data are being processing
+#     * how the code is optimized to run fast and be more concise
+
+# # Do not edit this cell! (20 points)
+# 
+# The notebook will also be checked for overall clean code requirements as specified at the **top** of this notebook. Some of these requirements include (review the top cells for more specifics): 
+# 
+# * Notebook begins at cell [1] and runs on any machine in its entirety.
+# * PEP 8 format is applied throughout (including lengths of comment and code lines).
+# * No additional code or imports in the notebook that is not needed for the workflow.
+# * Notebook is fully reproducible. This means:
+#    * reproducible paths using the os module.
+#    * data downloaded using code in the notebook.
+#    * all imports at top of notebook.
+
+# ## BONUS - Export a  .CSV File to Share (10 points possible)
+# 
+# This is optional - if you export a **.csv** file with the columns specified above: Site, Date and NDVI Value you can get an additional 10 points.
+# 
+# * FULL CREDIT: File exists in csv format and contains the columns specified.
+# We will check your github repo for this file!
+# 
+
+# In[13]:
+
+
+csv_dir = os.path.join(et.io.HOME, 'earth-analytics',
+                       'spring', 'hw', 'ea-2021-04-ndvi-automation-brianpinke')
+
+# Check path exists before trying to change directory, else make the directory
```

Comment:
Nice touch.

Reply:
Tbh I wasn't entirely sure if the path would exist in the CI or not so I did this out of an abundance of caution. I saw you were able to call the path without issue so that's good to know.